server/build_server.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
```

Comment:
it is a bit unclear where this script runs maybe define that with something like `cd $(dirname $(realpath $0))`